.github/workflows/deploy-to-k8s.yaml

```diff
@@ -7,6 +7,9 @@ on:
       environment:
         required: true
         type: string
+      useTag:
+        required: false
```

Comment:
I think it is required

Reply:
Actually I think I can just use the enviornment to do what I want here